com.playeveryware.eos/Runtime/Core/Config/EOSConfig.cs

```diff
@@ -441,6 +444,19 @@ public AuthScopeFlags GetAuthScopeFlags()
                 AuthScopeFlagsExtensions.TryParse);
         }
 
+        /// <summary>
+        /// Returns a single InputStateButtonFlags enum value that results from a
+        /// bitwise OR operation of all the defaultActivateOverlayButtonCombination flags on this
```

Comment:
The documentation should be updated to reflect the new name for the field member.

Reply:
Oh, yup! In fact that's why I normally use the <seealso cref="toggleFriendsButtonCombination"/>, it makes things auto change when I use that rename functionality. I added it now, here.